module/actor/sheet/actor-sheet.js

```diff
@@ -29,9 +29,58 @@ export default class PBActorSheet extends (foundry.appv1?.sheets?.ActorSheet ??
   constructEffectLists(sheetData) {
     const effects = {};
 
-    effects.temporary = sheetData.actor.effects.filter((i) => i.isTemporary && !i.disabled && !i.isCondition);
-    effects.disabled = sheetData.actor.effects.filter((i) => i.disabled && !i.isCondition);
-    effects.passive = sheetData.actor.effects.filter((i) => !i.isTemporary && !i.disabled && !i.isCondition);
+    // Helper function to add modifier display to effects
+    const addModifierDisplay = (effect) => {
+      const modifiers = [];
+      effect.changes.forEach(change => {
+        if (change.key.includes('combat.')) {
```

Comment:
Should this be the following? Or are there are effect keys that don't need the replacement?
```suggestion
        if (change.key.includes('system.attributes.combat.')) {
```

Reply:
I think youre right! looks like a typo on my end. Ill double check